### ISSUE TITLE
api.qub.ca should probably not be blocked

### ISSUE THREAD

**Issue by ppb2020** [StevenBlack/hosts#2732]
The addition of api.qub.ca on September 17 causes the TVA+ apps to no longer work or crash.

It's not clear why this would have been added as a bad site. Even if it might collect personal information or supply ads, it also provides a wanted service.

**Comment by StevenBlack** [comment]
Merci @ppb2020!

I have removed domain `api.qub.ca` but I'll keep an eye on it.  I added this domain after surfing on `journaldequebec.com` and I noted quite an avalanche of requests and `api.qub.ca` was a notable part of it.  Let's see how it goes after the next release.

BTW, it doesn't matter if a site provides a "wanted service".  We're all about slashing tracking, ads, or worse.

Closing.

**Comment by ppb2020** [comment]
> Merci @ppb2020!
> 
> I have removed domain `api.qub.ca` but I'll keep an eye on it. I added this domain after surfing on `journaldequebec.com` and I noted quite an avalanche of requests and `api.qub.ca` was a notable part of it. Let's see how it goes after the next release.
> 
> BTW, it doesn't matter if a site provides a "wanted service". We're all about slashing tracking, ads, or worse.
> 
> Closing.

Thank you for the quick response. This is appreciated. Quite interesting that you were consulting the Journal de Québec. :-)

The affected app is the TVA+ streaming application, which becomes non-functional (either crashes or unable to connect, depending on the platform) as a result of being unable to access the site. This means that anyone using your list (such as PiHole, as I would assume the use your list or similar sources) and who has no idea how this works, would see their app fail and be unable to use the TVA+ streaming service at all. It would not be obvious what the root cause of the failure is, unless they have the ability to debug the list or app. Completely blocking a service for no visible reason (to the naive users) is not likely something that you wish to see happen.

I agree with you and, obviously, support the goal of stopping tracking, ads, etc. and this is exactly why I use it myself. I have had issues in the past where a banking website I was using was failing because of their reliance on an ad-service site, which is not something that should break their site. I did manage to reach out to them and have them fix their site to be more resilient. I will certainly try this with TVA and see how receptive they are to suggestions on how to improve their app (it certainly should not crash because it cannot access api.qub.ca) to at least report more clearly what the issue is.

As to my use of the list, it is something I wrote myself and I can easily adapt my code to deal with a list of exceptions for sites that I wish to nonetheless access, understanding the potential risk of doing so,

Thanks again for the quick response and all the work you do to keep the list up to date, a never ending process.

Sincerely.
